app/controllers/submissions_controller.rb

```diff
@@ -33,6 +33,8 @@ class SubmissionsController < ApplicationController
                        :populate_peer_submissions_table]
   before_filter :authorize_for_user, only: [:download, :downloads]
 
+  layout 'assignment_content'
```

Comment:
Let's not change it globally for all routes here. Instead, please use the layout specifically for the actions you changed (option to `render`).

Reply:
I'm happy to do this.  Does it make sense then to add the layout option to **all** calls to `render: :file_manager`? Specifically, would you recommend changing the two calls in assignment_controller.rb (lines 732 and 810) to also use `layout: 'assignment_content'` as a keyword argument to render?

Reply:
@david-yz-liu . I agree that this shouldn't be changed for all routes but the problem is if I only pass the layout as an keyword argument when I call render, then flash messages are not rendered properly if this page is not displayed through a call to `render` (ie. by navigating to `.../assignments/X/submissions/file_manager`). 
What I would suggest instead is changing this line to 

    layout 'assignment_content', only: [:file_manager]

That way this page will be displayed with the assignment_content layout by default **unless** the `layout` keyword argument is used when calling render.